app/src/main/res/layout-land/administrator_controls_account_actions_view.xml

```diff
@@ -18,14 +18,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
`android:layout_height="wrap_content"` can be removed as it is already defined in `Caption -> TextView.Common1`

Reply:
@rt4914 this PR is for `administrator_controls_app_information_view.xml` and not for `administrator_controls_account_actions_view.xml`.  
Hence, shouldn't we revert changes in each `administrator_controls_account_actions_view.xml` file and only leave those in `administrator_controls_app_information_view.xml` files?

Reply:
@MohamedMedhat1998 Yes you are right. Lets do one thing, file an issue for `administrator_controls_account_actions_view.xml` and link it here. Meaning we will be fixing two issues in this PR.

Reply:
@MohamedMedhat1998 PTAL https://github.com/oppia/oppia-android/pull/1992#issuecomment-712203336